backend/python/app/resources/entity_dto.py

```diff
@@ -1,3 +1,14 @@
+# file-storage {
+ALLOWABLE_CONTENT_TYPES = [
+    "text/plain",
+    "application/pdf",
+    "image/png",
+    "image/jpeg",
+    "image/gif",
+]
+# } file-storage
+
+
```

Comment:
Should include spaces in `# } file-storage`, otherwise no file storage copy will have 2 spaces at the top.